### PR TITLE
Tests with mocked requests() method. 

### PR DESCRIPTION
**Pull request by trjaffe**

### CODE REVIEW

**Comment by trjaffe** [comment]
FYI, just realized the mocked results are not working at all. Still working on it.

**Comment by trjaffe** [comment]
OK, it should be working now. (I seriously have to unplug my ethernet to be sure it's using the mocked bits.)

File: astroquery/vo/tests/thetests.py [hunk 1]
```diff
@@ -0,0 +1,262 @@
+import json, os, pandas
```
Comment (tomdonaldson) [inline astroquery/vo/tests/thetests.py]: I'm a little concerned about the pandas dependency here.  Can we serialize the tables just using built-in astropy features.  It looks like reading/writing to/from the ascii ecsv format is supposed to support lossless round-tripping.

File: astroquery/vo/tests/thetests.py [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+## Why can't I import astroquery.vo ?
+from astroquery.vo import Registry
+
+##
+##  Regenerate the output JSON files using:
+##
+##  astroquery/vo > python tests/thetests.py
+
+
+def table2json_old(current, fname, suffix='.json'):
```
Comment (tomdonaldson) [inline astroquery/vo/tests/thetests.py]: Should these *_old functions be removed?

**Comment by tomdonaldson** [comment]
Looks good now.

**Comment by tomdonaldson** [comment]
looks good now

**Pull request merged**